bin/dokku-daemon

```diff
@@ -129,20 +115,8 @@ connection() {
             response="Requires --force flag"
             status=1
           else
-            if [[ " ${cmd[-2]} " == *"<"* ]]; then # Check if stdin
-              file_path=${cmd[-1]}
-              unset "cmd[${#cmd[@]}-1]" # remove last element which is file path
-              unset "cmd[${#cmd[@]}-1]" # remove last element which is <
-              response=$(flock "$DOKKU_LOCK_PATH" dokku "${cmd[@]}" < ${file_path} 2>&1)
-            elif [[ " ${cmd[-2]} " == *">"* ]]; then  # Check if stdout
-              file_path=${cmd[-1]}
-              unset "cmd[${#cmd[@]}-1]" # remove last element which is file path
-              unset "cmd[${#cmd[@]}-1]" # remove last element which is >
-              response=$(flock "$DOKKU_LOCK_PATH" dokku "${cmd[@]}" > ${file_path} 2>&1)
-            else
```

Comment:
This removes stdin/stdout?

Reply:
I had to make many changes and pull requests recently related to dokku and letsencrypt both, so I cannot remember the reasoning behind this change.

I will revert it and test to verify functionality.